### ISSUE TITLE
tours.html image is missing alt attribute

### ISSUE THREAD

**Issue by rseyum** [rseyum/github-issue-templates#6]
**Describe the bug**
The tours.html is missing the image alt attribute this can confuse the NVDA or any screen reader for visually disabled people.

**To Reproduce**
1. Go on tours.html and check the fast pass 
2. the fast pass result will suggest possible fixes.
3. right-click on the page and inspect the element to try to fix the issue by adding alt and description to the image.
4. check the FastPass issue
5. check the page again

**Expected behavior**
the image should have alt and should pass the fast pass check with 0 error.

**Screenshots**
![accessibility1](https://user-images.githubusercontent.com/103169939/172504160-38a2fbe4-9a89-44af-98eb-ea142742e497.PNG)

**Desktop (please complete the following information):**
N/A

**Smartphone (please complete the following information):**
N/A

**Additional context**
This fix affects screen readers too.
